### ISSUE TITLE
Drop support for FMI 1.0

### ISSUE THREAD

**Issue by OpenModelica-TracImporter** [OpenModelica/OpenModelica#6143]
We don't support FMI 1.0 (export) well. Are we even testing FMI 1.0?
And there are hardly any people using FMI 1.0 export.
OMSimulator won't import FMI 1.0 either and we already stopped developing the OpenModelica Modelica import of FMUs.

So should we remove source code that is used only for FMI 1.0.
That could simplify the FMI runtime, the code generation and our build system and remove dead code.

What are your comments on this topic? Are you in favor or cleaning up the source code or keep what ever functionality we have for someone who want's to use it?

----

Migrated-From: https://trac.openmodelica.org/OpenModelica/ticket/6143

**Comment by AnHeuermann** [comment]
> Just testing is not a good enough reason 😅

Yes, so we should actually remove FMI 1.0 from OpenModelica. Currently users can still call FMI 1 export (and then it fails).

**Comment by casella** [comment]
Agreed :)

So, 
- [buildModelFMU](https://github.com/modelica/ModelicaSpecification/issues) should report an error when called with `version = "1.0"`
- OMEdit Tools | Options | FMI should only mention 2.0

@adeas31 could you take care of that?

**Comment by AnHeuermann** [comment]
Also we should remove FMI1 from OMCompiler and the runtimes. I started that, but it is challenging because FMI and FMI1 are not separated everywhere. See  https://github.com/AnHeuermann/OpenModelica/tree/remove-fmi-1.
